dev-manuals/repositories/add-license-notices-at-pr.md

```diff
@@ -0,0 +1,8 @@
+# How to check, add and update license notices at every PR
+
+If you add or update a dependency it must be added to the third party license notice file before tomorrow.
+
+To do that you have to add a new line in this style `{"licTypeName": "MIT", "compName": "UniTask", "compVersion": "2.3.1", "copyright": "Copyright (c) 2019 Yoshifumi Kawai / Cysharp, Inc.", "copyrightSource": "https://github.com/Cysharp/UniTask/blob/2.3.1/LICENSE"}` to [this file](https://github.com/Gamify-IT/third-party-license-notice/blob/main/LicenseNotes.json).
```

Comment:
````suggestion
To do that, you should add a new line like
```json
{"licTypeName": "MIT", "compName": "UniTask", "compVersion": "2.3.1", "copyright": "Copyright (c) 2019 Yoshifumi Kawai / Cysharp, Inc.", "copyrightSource": "https://github.com/Cysharp/UniTask/blob/2.3.1/LICENSE"}
```
to <https://github.com/Gamify-IT/third-party-license-notice/blob/main/LicenseNotes.json>.
````

Reply:
Ahem, please rename the json file?
I'd like to use kebab-case filenames where possible?

Reply:
As you've told me, the filenames appear to be convention as well.
In that case, leave them as they are.